CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 # Release Notes
 
+## Version 1.29.0 - 2023-01-26
+
+Features:
+* Graduate QEMU driver from experimental to GA [#15556](https://github.com/kubernetes/minikube/pull/15556)
+* Implement --static-ip flag (Docker only) [#15553](https://github.com/kubernetes/minikube/pull/15553)
+* Implement Kicbase/ISO version validation when starting existing clusters [#15235](https://github.com/kubernetes/minikube/pull/15235)
```

Comment:
this should be under improvements, wording should say
"Warn user when using old ISO and base image"